### PR TITLE
Mac: Application should handle reopen

### PR DESCRIPTION
**Pull request by erikjv**

### CODE REVIEW

File: src/gui/cocoainitializer_mac.mm [hunk 1]
```diff
@@ -12,28 +12,51 @@
...
+- (BOOL)applicationShouldHandleReopen:(NSApplication *)sender hasVisibleWindows:(BOOL)flag;
+@end
+
+@implementation OwnAppDelegate {
+}
+
+- (BOOL)applicationShouldHandleReopen:(NSApplication *)sender hasVisibleWindows:(BOOL)flag
+{
+    if (auto app = dynamic_cast<OCC::Application *>(QApplication::instance()))
+        app->showSettingsDialog();
```
Comment (TheOneRing) [inline src/gui/cocoainitializer_mac.mm:33]: Hmm qobject_cast?
Nitpick I prefer {} for all if's

File: translations/client_ar.ts [hunk 2]
```diff
@@ -76,18 +76,26 @@
 <context>
```
Comment (TheOneRing) [inline translations/client_ar.ts:76]: Could you rebase on master without the translations commit :)

**Comment by TheOneRing** [comment]
Could you also add changelog entry like that one :)
https://github.com/owncloud/client/blob/master/changelog/unreleased/4542

File: translations/client_ar.ts (hunk 2 above)
Comment (dschmidt) [reply translations/client_ar.ts:76]: that will also make CLA-Assitant happy

File: src/gui/cocoainitializer_mac.mm (hunk 1 above)
Comment (erikjv) [reply src/gui/cocoainitializer_mac.mm:33]: done

File: changelog/unreleased/8812 [hunk 3]
```diff
@@ -0,0 +1,7 @@
+Bugfix: Relaunching the client on macOS will show the settings dialog
+
+Relaunching the ownCloud client when it is already running, would
+seemingly do nothing at all. To make this more consistent with other
+macOS applications, relaunching will now open the settings dialog.
+
+https://github.com/owncloud/client/pull/8812
```
Comment (TheOneRing) [inline changelog/unreleased/8812:7]:
``` suggestion
https://github.com/owncloud/client/issues/8377
https://github.com/owncloud/client/pull/8812

**Review by TheOneRing — APPROVED** [review]

**Pull request merged**